deel-2-flexbox-layout/opdracht-1/styles/styles.css

```diff
@@ -1 +1,45 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+
+body {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+header {
+    display: flex;
+    flex-basis: 100px;
+    flex-shrink: 0;
+    flex-grow: 0;
+}
+
+main {
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+    flex-shrink: 0;
+}
+
+article {
```

Comment:
Het werkt op deze manier ook alleen in de uitwerkingen word bij zowel het article, nav en aside een order gedeclareerd